inc/saineighbor.h

```diff
@@ -115,6 +115,17 @@ typedef enum _sai_neighbor_entry_attr_t
      */
     SAI_NEIGHBOR_ENTRY_ATTR_COUNTER_ID,
 
+    /**
+     * @brief Remote Out Logical Interface
+     *
+     * Defines the remote neighbor interface when the destination is on a remote line card
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0
```

Comment:
we should add @validonly in general for all VOQ based attributed

Reply:
Will add